packages/babel-node/src/_babel-node.js

```diff
@@ -39,6 +39,7 @@ program.option(
   "List of extensions to hook into [.es6,.js,.es,.jsx,.mjs]",
   collect,
 );
+program.option("--source-type [script|module|unambiguous]", "");
```

Comment:
can this be made to assume "module" for `.mjs` files?

Reply:
`.mjs` files are hard-coded to `module` currently, so this would only effect other types.

Reply:
Thanks, makes sense.